modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/editor/config/DefaultBalloonEditorConfiguration.js

```diff
@@ -50,7 +50,7 @@ const DEFAULT_BALLOON_EDITOR_CONFIG = {
 	toolbarLink: 'LinkRemove,LinkTarget,LinkInput,LinkConfirm,LinkBrowse',
 	toolbarTable: 'TableHeaders,TableRow,TableColumn,TableCell,TableDelete',
 	toolbarText:
-		'Styles,Bold,Italic,Underline,BulletedList,NumberedList,TextLink,JustifyLeft,JustifyCenter,JustifyRight,JustifyBlock,LineHeight,BGColor,RemoveFormat',
+		'Styles,Bold,Italic,Underline,BulletedList,NumberedList,TextLink,Twitter,JustifyLeft,JustifyCenter,JustifyRight,JustifyBlock,LineHeight,BGColor,RemoveFormat',
```

Comment:
Is Twitter button intended to be a default in BalloonToolbar?

Reply:
Nope. I think it's only in Blogs.

Reply:
We should probably have a custom configuration for the sample module. It can end up showcasing all plugins and config setting specific for all usages.